config/config.go

```diff
@@ -127,3 +136,17 @@ func GetApiServiceAddressWithDefault(config Config) string {
 	}
 	return fmt.Sprintf("http://localhost:%v", config.Api.Port)
 }
+
+func GetSignalWithStartOnWithDefault(config Config) string {
+	if config.Api.WaitForStateCompletionMigration.SignalWithStartOn != "" {
+		return config.Api.WaitForStateCompletionMigration.SignalWithStartOn
+	}
+	return "old"
+}
+
+func GetWaitForOnWithDefault(config Config) string {
```

Comment:
Nit: I actually should have done it this using struct member method instead:
```
func (c Config) GetSignalWithStartOnWithDefault() string{
    // the same code here
}
```
And to use it:
```
config.GetSignalWithStartOnWithDefault()
```

This is very common pattern in Golang, I don't know why I didn't do that way 😂

Reply:
Done